### PR TITLE
gdb: Update to 10.1

### PR DESCRIPTION
**Pull request by saurabhchardereal**

### CODE REVIEW

File: packages/gdb/build.sh [hunk 1]
```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://www.gnu.org/software/gdb/
 TERMUX_PKG_DESCRIPTION="The standard GNU Debugger that runs on many Unix-like systems and works for many programming languages"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=9.2
-TERMUX_PKG_REVISION=3
+TERMUX_PKG_VERSION=10.1
+TERMUX_PKG_REVISION=1
```
Comment (Grimler91) [inline packages/gdb/build.sh:6]: TERMUX_PKG_REVISION can be removed completely when version is increased (REVISION is set to 0 per default)

**Review by Grimler91 — COMMENTED** [review]
@saurabhchardereal have you tested it on device on some arch, just to verify that gdb starts and breaks and backtraces still work?

File: packages/gdb/build.sh (hunk 1 above)
Comment (saurabhchardereal) [reply packages/gdb/build.sh:6]: oops. Thanks for pointing out!

**Comment by Grimler91** [comment]
Great, nice work!

**Pull request merged**